.github/workflows/mirror-images-regsync.yml

```diff
@@ -34,9 +34,9 @@ jobs:
           export PATH=$PATH:$(pwd)
           time regsync once --missing --config regsync.yaml
         env:
```

Comment:
Technically I think you can remove all these...since it's already in the global env after the EIO read step. This doesn't break anything but is likely redundant unless renaming ENVs for this particular step.